### ISSUE TITLE
When making Windows 10 images, install.wim identifies most SKUs as Windows 11

### ISSUE THREAD

**Issue by XenoPanther** [gus33000/UUPMediaCreator#125]
I just finished making a 18362.1 ISO and noticed that most SKUs report "Windows 11"
![image](https://user-images.githubusercontent.com/81916167/175054243-227402eb-7a39-486a-9cfc-12be58ed4373.png)

**Comment by gus33000** [comment]
Yeah this would require dynamically reading the sku name from basebrd, something I haven't figured out how to exactly do yet because it's going through a mess of NtQuerySystemInformation calls and the ids in the mui don't match the sku id itself (and differ between server/client). You can always edit the Settings.ini file for now to correct the names.

**Comment by stdin82** [comment]
> 

Why not read or check the base image name (Home/Pro)?
or just assume that any > 21900 build is Windows 11

**Comment by gus33000** [comment]
Some skus don't use client branding and the part after Windows XX also needs to match the current branding as well. But I agree this could be a good solution for now
